core/store/migrate/migrations/0202_default_values_for_last_finalized_block.sql.sql

```diff
@@ -0,0 +1,33 @@
+-- +goose Up
+
+WITH variables AS (
+    SELECT
+        evm_chain_id,
+        CASE
+            WHEN evm_chain_id = 43113 then 1 -- Avax Fuji
+            WHEN evm_chain_id = 43114 then 1 -- Avax Mainnet
```

Comment:
might be worth including https://github.com/smartcontractkit/ccip/blob/ccip-develop/core/chains/evm/config/toml/defaults/Avalanche_ANZ_testnet.toml#L2 as well, though its just a testnet so not a big deal.